examples/timeline/groups/groupsEditable.html

```diff
@@ -55,7 +56,30 @@
 	{"content": "WEC", "id": "WEC", "value": 18, className:'endurance'},
 	{"content": "GP2", "id": "GP2", "value": 19, className:'openwheel'}
   ]);
-  
+
+  // loop through groups and convert content strings into elements with hide buttons.
+  groups.forEach(function(group){
```

Comment:
You're better off using the `options.groupTemplate` to achieve this. This seems kind of hacky and unclear...